src/main/kotlin/no/nav/syfo/client/person/oppfolgingstilfelle/OppfolgingstilfellePerson.kt

```diff
@@ -2,7 +2,12 @@ package no.nav.syfo.client.person.oppfolgingstilfelle
 
 import java.time.LocalDate
 
+const val ARBEIDSGIVERPERIODE_DAYS = 16L
+
 data class OppfolgingstilfellePerson(
     val fom: LocalDate,
     val tom: LocalDate,
 )
+
+fun OppfolgingstilfellePerson.isInactive(): Boolean =
```

Comment:
Hvorfor er kriteriet 16 dager her? (i kommentaren til PR'en skriver du om en 4-måneders regel).

Reply:
Satt den til 16-dager fordi da er personen sannsynligvis ikke syk mer.
4 måneder er gammel en sykmelding kan være for at personen dukker i visningen til nærmeste leder.
Dvs. at om man f.eks  tillott å innkalle til møter opptil 4 måneder etter aktiv tilfelle, så ville man risikert at det blir innkalt, men at man andre får beskjed om andre tilstandsendringer som f.eks avlysning.

Med denne regelen vil man hindre at man kaller inn personer som ikke er syke og en veileder vil har 4 måneder på seg til å fullføre en inkallings-pross dersom personen slutter å være syk eller går over på AAP.